src/cartogram_info/write_geojson.cpp

```diff
@@ -193,7 +193,7 @@ void CartogramInfo::write_geojson(
     combined_json["Original"] = new_json_original;
     std::cout << combined_json << std::endl;
```

Comment:
Should we also include the `equal-area-map` as we would require it in the future on the website?

Reply:
In a future pull-request, @atima can tack that up as an issue when needed.